src/components/craft/CraftBoxModelInput.tsx

```diff
@@ -26,7 +26,11 @@ const CraftBoxModelInput: React.VFC<{
           onChange={e => {
             const eventBoxModelValue = formatBoxModelValue(e.target.value)
             onChange?.(
-              `${eventBoxModelValue?.[0]};${eventBoxModelValue?.[1]};${eventBoxModelValue?.[2]};${eventBoxModelValue?.[3]}`,
+              `${isNaN(eventBoxModelValue?.[0] || 0) ? eventBoxModelValue?.[0] || 0 : 0};${
+                isNaN(eventBoxModelValue?.[0] || 0) ? eventBoxModelValue?.[1] || 0 : 0
+              };${isNaN(eventBoxModelValue?.[0] || 0) ? eventBoxModelValue?.[2] || 0 : 0};${
+                isNaN(eventBoxModelValue?.[0] || 0) ? eventBoxModelValue?.[3] || 0 : 0
+              }`,
```

Comment:
how about use `range()` in Ramda.js

```js
range(0, 4).map(index => isNaN(eventBoxModelValue?.[index] || 0) ? eventBoxModelValue?.[index] || 0 : 0).join(';')
```

btw, we can discuss this code in the code review to optimize the readibility

FYR: https://ramdajs.com/docs/#range